sympy/simplify/trigsimp.py

```diff
@@ -1183,7 +1183,7 @@ def _futrig(e):
     else:
         coeff = None
 
-    Lops = lambda x: (L(x), x.count_ops(), _nodes(x), len(x.args), x.is_Add)
+    Lops = lambda x: (L(x), measure(x) if measure is not None else x.count_ops(), _nodes(x), len(x.args), x.is_Add)
```

Comment:
It would be better to define this measure function once above rather than have the if check inside the lambda function (that will be called many times).